cypress/support/pages/checout.pageObject.js

```diff
@@ -0,0 +1,52 @@
+import PageObject from '../PageObject';
+
+class ChecoutPageObject extends PageObject {
```

Comment:
The class name 'ChecoutPageObject' seems to be misspelled. It's recommended to use correct spelling to avoid confusion. Did you mean 'CheckoutPageObject'?

Reply:
Yes, it was a typo